src/components/icon/icon.tsx

```diff
@@ -65,15 +65,7 @@ export class Icon {
      * will remain the way they look in the file.
      * Therefore we inject the svg as inline markup instead.
      */
-    private renderSvg(svgData: string, name: string) {
-        const container = this.host.shadowRoot.querySelector('div.container');
-        if (container) {
-            container.innerHTML = svgData;
-        } else {
-            // tslint:disable-next-line:no-console
-            console.error(
-                `limel-icon: no container found when trying to render svg! icon: ${name}`
-            );
-        }
+    private renderSvg(svgData: string) {
+        this.host.shadowRoot.innerHTML = svgData;
```

Comment:
This bypasses the problem completely 🙂

Reply:
Unfortunately this will not work :slightly_frowning_face: This will overwrite *all* the elements inside the host, including the `<style>` tag generated by Stencil. If we did not have any styles it would be fine though :smile:

Reply:
Ah crap… good catch!

Reply:
![image](https://user-images.githubusercontent.com/154725/77566701-5234e680-6ec6-11ea-8081-baa6011c202d.png)

😂😂😂

Reply:
⚡️